src/utils/index.js

```diff
@@ -293,67 +293,60 @@ export const doesPlotContainCrop = plot =>
   plot !== null && getPlotContentType(plot) === itemType.CROP
 
 export const getLifeStageRange = memoize((
-  /** @type {farmhand.cropTimetable} */ cropTimetable
+  /** @type {farmhand.cropTimeline} */ cropTimeline
 ) => {
-  return [SEED, GROWING].reduce(
-    /**
-     * @param {farmhand.cropLifeStage[]} acc
-     */
-    (acc, stage) => {
-      if (stage === GROWING) {
-        return acc.concat(
-          cropTimetable[stage].reduce(
-            (acc2, value) => acc2.concat(Array(value).fill(stage)),
-            []
-          )
-        )
-      }
+  let lifeStageRange = Array(cropTimeline.slice(0, 1)[0]).fill(SEED)
```

Comment:
Could this be simplified to something like the following?

```suggestion
  let lifeStageRange = Array(cropTimeline[0]).fill(SEED)
```

Reply:
oh man why did i overcomplicate this so much haha